Autor: Alessio Treglia <quadrispro@ubuntu.com>
Description: Plugins have been renamed, fix the search path.
Bug: http://bugs.debian.org/581714
---
 source/main.cc |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- aeolus.orig/source/main.cc
+++ aeolus/source/main.cc
@@ -193,8 +193,8 @@ int main (int ac, char *av [])
 
     if (mlockall (MCL_CURRENT | MCL_FUTURE)) fprintf (stderr, "Warning: memory lock failed.\n");
 
-    if (t_opt) sprintf (s, "%s/aeolus_txt.so", LIBDIR);
-    else       sprintf (s, "%s/aeolus_x11.so", LIBDIR);
+    if (t_opt) sprintf (s, "%s/aeolus_txt.so.0", LIBDIR);
+    else       sprintf (s, "%s/aeolus_x11.so.0", LIBDIR);
     so_handle = dlopen (s, RTLD_NOW);
     if (! so_handle)
     {
